source/reference/method/MongoDBCollection-find.txt

```diff
@@ -52,9 +52,15 @@ Parameters
 
      * - batchSize
        - integer
-       - The number of documents to return in the first batch. Defaults to
-         ``101``. A batchSize of ``0`` means that the cursor will be
-         established, but no documents will be returned in the first batch.
+       - The maximum number of documents to return in the first batch. By default, the ``find()`` 
+         command has an initial batch size of ``101`` documents. This
```

Comment:
This should definitely specify:

> ...and a maximum size of 16 mebibytes for each subsequent batch

As-is, it's inconsistent with the other docs snippets and could mislead users into thinking this has no effect on `getMore` for subsequent batches.